<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPBYMINW1873" tests="9" failures="5" timestamp="20 Dec 2017 07:37:05 GMT" time="0.026" errors="0">
  <testcase name="SubtractionVerivfication" time="0.006" classname="CalculatorSubTest"/>
  <testcase name="SubtractionVerivfication" time="0.0" classname="CalculatorSubTest"/>
  <testcase name="SubtractionVerivfication" time="0.0" classname="CalculatorSubTest"/>
  <testcase name="TgVerivfication" time="0.007" classname="CalculatorTgTest">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [0.0]
	at CalculatorTgTest.TgVerivfication(CalculatorTgTest.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TgVerivfication -->
  <testcase name="TgVerivfication" time="0.0" classname="CalculatorTgTest">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [-3.380515]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [-3.380515]
	at CalculatorTgTest.TgVerivfication(CalculatorTgTest.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TgVerivfication -->
  <testcase name="TgVerivfication" time="0.0" classname="CalculatorTgTest">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [0.0]
	at CalculatorTgTest.TgVerivfication(CalculatorTgTest.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TgVerivfication -->
  <testcase name="TgVerivfication" time="0.0" classname="CalculatorTgTest">
    <failure type="java.lang.AssertionError" message="expected [NaN] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [NaN] but found [0.0]
	at CalculatorTgTest.TgVerivfication(CalculatorTgTest.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TgVerivfication -->
  <testcase name="TgVerivfication" time="0.0" classname="CalculatorTgTest"/>
  <testcase name="TgVerivfication" time="0.0" classname="CalculatorTgTest">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [0.36985699966737745]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [0.36985699966737745]
	at CalculatorTgTest.TgVerivfication(CalculatorTgTest.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TgVerivfication -->
</testsuite>
